### ISSUE TITLE
x64 version can not show utf8 character

### ISSUE THREAD

**Issue by youxiachai** [rhiokim/haroopad#443]
I instal 0.13 win x64 ,It can't show utf8 character, but ia32 not this bug

**Comment by rhiokim** [comment]
Could you please explain more detail or provide a simple case reproducing this?

**Comment by youxiachai** [comment]
copy to text and save file , ia32 show well, but x64 can't

```
：香辣鸡翅+煎蛋：香辣鸡翅+煎蛋：香辣鸡翅+煎蛋：香辣鸡翅+煎蛋：香辣鸡翅+煎蛋：香辣鸡翅+煎蛋 xxxxx
```

**Comment by rhiokim** [comment]
New version is out. v0.13.1 
Could you please check this issue.

**Comment by youxiachai** [comment]
@rhiokim x64 version editor window not show , but preview windows show 

here is capture
![a642](https://cloud.githubusercontent.com/assets/929502/6704456/fc3fb766-cd83-11e4-805c-b79fa8dc2487.png)

**Comment by youxiachai** [comment]
also perference have bug
![a10](https://cloud.githubusercontent.com/assets/929502/6704473/2e2ea278-cd84-11e4-8050-3da233e0de84.png)

**Comment by rhiokim** [comment]
Hi there

I've made new version package for next release. it's release candidate version, not official version
https://bitbucket.org/rhiokim/haroopad-download/downloads

I had tried to solve this issue many time. But I did not solve yet. Because I can't get same environment. When I changed my system local from Korea to China. At this time, it's work fine.

Could I get your feedback about this, If you have spare time for Haroopad?
Thanks a lot.
